### ISSUE TITLE
Can't bind button to keystroke

### ISSUE THREAD

**Issue by Hjdskes** [libratbag/libratbag#594]
This was reported in https://github.com/libratbag/piper/issues/242, but I believe this is an issue in libratbag.

**Comment by dissyllabic** [comment]
Hi, I am still having this issue with latest fedora (f29) installed from dnf. Using a Logitech G403 over the actual cable (Piper crashes when I try to apply with just dongle, apparently because of a sync/async issue that was solved (?)

**Comment by WindFreaker** [comment]
I think I am having a similar issue? Or maybe its the same issue (I don't fully understand the original bug report on piper).

If (using piper) I attempt to bind `<ctrl> A` to a mouse button, it will work. If I then try to bind just `S` to the same mouse button, I effectively end up with the bind `<ctrl> S` instead. If I then try to bind `<shift> H`, it works properly and pressing the mouse button gives me a capital H. If I then try to bind just `G` pressing that mouse button will print capital Gs and not lowercase.

Basically all I want to do is bind a singular keystroke to a mouse button, but it seemingly **HAS** to have a modifier as well.

I am using the G305 and cannot test using a wire. This is with the dongle. I am on Fedora 42.
